packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -57,60 +57,65 @@ describe('background sync queue test', () => {
     await backgroundSyncQueue.cleanupQueue();
   });
 
-  it('check defaults', () => {
+  it('should get default values if nothing is given in constructor', () => {
```

Comment:
It's not clear to me from this `describe`/`it` block what this testing.

I prefer to nest tests as @jeffposnick described in https://github.com/GoogleChrome/workbox/issues/705#issuecomment-318675972. Here's what I'd do in this case:

```js

describe(`BackgroundSyncQueue`, function() {
  describe(`constructor`, function() {
    it(`should use default values when nothing is passed`, function() {
      // ...
    });
  });

  describe(`someOtherMethod`, function() {
    it(`should do something the method does`, function() {
      // ...
    });
  });
});
```